roles/rsyslog/vars/outputs/files/main.yml

```diff
@@ -19,11 +19,15 @@ __rsyslog_conf_files_output_modules:
     type: 'modules'
     sections:
 
+      # yamllint disable rule:line-length
       - comment: 'Log messages into files with timestamp format either
-        traditional or default'
+        traditional, syslog or default'
         options: |-
-          {% if rsyslog_basics_use_traditional_timestamp_format | d(true) %}
```

Comment:
is `rsyslog_basics_use_traditional_timestamp_format` still used?

Reply:
No, not any more. I decided to use the newly introduced param `logging_files_template_format` for the purpose instead. (Please note that `rsyslog_basics_use_traditional_timestamp_format` was not in the doc. so, I'd think it had no chance to be set...)